src/Raven.Server/RavenServer.cs

```diff
@@ -237,6 +240,13 @@ void ConfigureKestrel(KestrelServerOptions options)
                         // the .Wait() can throw as well, so we'll ignore any
                         // errors here, it all goes to the log anyway
                     }
+
+                    var httpsUrl = new Uri(WebUrl);
```

Comment:
No, this binds the public URL, no?
We need to bind to the same sources that with bind on 443 on.